CONTRIBUTING.md

```diff
@@ -130,7 +137,7 @@ A pull request doesn’t have to represent finished work. We always recommend op
 - [Open Source Guide: “How to Contribute to Open Source”](https://opensource.guide/how-to-contribute/)
 - [Website](https://quire.getty.edu)
 - [Issue Tracking](https://github.com/thegetty/quire-docs/issues/)
-- [Documentation](https://quire.getty.edu/documentation)
+- [Documentation](https://quire.getty.edu/docs-v1)
```

Comment:
This is in a list of "Helpful Resources" for contributing to the docs, which maybe means it should be a link to the legacy/hugo docs since that's what they'll be working in?